components/library/librsvg/pkg5

```diff
@@ -14,6 +14,7 @@
         "system/library/fontconfig",
         "system/library/freetype-2",
         "system/library/gcc-10-runtime",
+        "system/library/gcc-13-runtime",
```

Comment:
Why depends this on both gcc-10 and gcc-13 runtimes?

Reply:
I guess it is a race condition, because the installed librsvg package depends on gcc-10 runtime and if you check on build/prototype/i386/usr/lib/amd64/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so it loads /usr/lib/amd64/librsvg-2.so.2 which is depended on gcc-10. Unfortunately I can't uninstall the old librsvg package due to many dependencies.

Reply:
Add pkg.depend.bypass-generate=.* in the manifest for every affected file. The rerun gmake REQUIRED_PACKAGES and gmake publish. Here it was necessary to change two lines (I have only checked the first):
file path=usr/bin/rsvg-convert pkg.depend.bypass-generate=.*
file path=usr/lib/$(MACH64)/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so pkg.depend.bypass-generate=.*

Reply:
I am not sure whether this is a good idea. Maybe it would be better to first merge as is and then re-publish revision 1. If your theory is sound this would fix the dependency problem.

Reply:
No, this doesn't bring the desired result. Even after installation of the new version gmake REQUIRED_PACKAGES still produces the dependency on gcc-10. There must be something else needing it.